### PR TITLE
New tools branch

### PR DESCRIPTION
**Pull request by codecat22**
Changes from tools minus the battery, which is kept as is on main. Contains blender files, two switch models, textures, and two prefabs for each (base and no base).

### CODE REVIEW

File: Connected/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -691,7 +691,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 0
+  activeInputHandler: 1
```
Comment (steenish) [inline Connected/ProjectSettings/ProjectSettings.asset]: This is not appropriate, this should be 0

**Review by steenish — CHANGES_REQUESTED** [review]
looks mostly good, but the new input system should not be used

File: Connected/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (codecat22) [reply Connected/ProjectSettings/ProjectSettings.asset]: > 

To make sure, is it enough to change the number in settings to 0? Or should something be altered in Unity as well?

Comment (steenish) [reply Connected/ProjectSettings/ProjectSettings.asset]: Yep, that should be enough

**Review by raaavioli — APPROVED** [review]
Lgtm

**Pull request merged**